source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -527,6 +527,48 @@ void Filter::sendTrailers(ProcessorState& state, const Http::HeaderMap& trailers
   stats_.stream_msgs_sent_.inc();
 }
 
+void Filter::onNewTimeout(
+    const std::unique_ptr<envoy::service::ext_proc::v3::ProcessingResponse>&& response) {
```

Comment:
std::unique_ptr already implies the ownership, I don't think you need the && here.

Reply:
Changed into passing a uint32_t number in.